### PR TITLE
Rails (Ruby on)

### PR DESCRIPTION
**Pull request by Crell**
Rails doesn't have an upstream, which makes it a bit easier to add but probably harder to maintain long term.  Poopy.

See: https://github.com/platformsh/template-rails/pull/1

### CODE REVIEW

File: templates/rails/files/.platform.app.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: "app"
```
Comment (otaviojava) [inline templates/rails/files/.platform.app.yaml]: Please, don't forget to put details on this file, e.g.:
https://github.com/platformsh/template-drupal8/blob/master/.platform.app.yaml

File: templates/rails/files/.platform/routes.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+"https://{default}":
```
Comment (otaviojava) [inline templates/rails/files/.platform/routes.yaml]: Please, don't forget to put details on this file, e.g.:
https://github.com/platformsh/template-drupal8/blob/master/.platform/routes.yaml

File: templates/rails/files/app/assets/config/manifest.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
```
Comment (otaviojava) [inline templates/rails/files/app/assets/config/manifest.js]: Why do we need this file?

Comment (Crell) [reply templates/rails/files/app/assets/config/manifest.js]: It was auto-generated by the Rails setup script.  I have no idea what it does, honestly.

File: templates/rails/files/app/models/application_record.rb [hunk 4]
```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```
Comment (otaviojava) [inline templates/rails/files/app/models/application_record.rb]: Active Record :feelsgood: The most controversial pattern ever :smiley:

Comment (Crell) [reply templates/rails/files/app/models/application_record.rb]: ¯\_(ツ)_/¯  It's what Rails uses natively and there's no meaningful alternative.

File: templates/rails/files/.platform/routes.yaml (hunk 2 above)
Comment (otaviojava) [inline templates/rails/files/.platform/routes.yaml]: Same here: https://github.com/platformsh/template-drupal8/blob/master/.platform/routes.yaml

File: templates/rails/files/.platform.app.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+#
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
+database: 'postgresqldb:postgresql'
+
+# The size of the persistent disk of the application (in MB).
+disk: 1024
+
```
Comment (otaviojava) [inline templates/rails/files/.platform.app.yaml]: too many lines, one is enough :)

**Comment by otaviojava** [comment]
@Crell I got this on the Ruby application

```
        Post-install message from sass:
        
        Ruby Sass has reached end-of-life and should no longer be used.
        
        * If you use Sass as a command-line tool, we recommend using Dart Sass, the new
          primary implementation: https://sass-lang.com/install
        
        * If you use Sass as a plug-in for a Ruby web framework, we recommend using the
          sassc gem: https://github.com/sass/sassc-ruby#readme
        
        * For more details, please refer to the Sass blog:
          https://sass-lang.com/blog/posts/7828841
        
        W: FATAL: Listen error: unable to monitor directories for changes.
        W: Visit https://github.com/guard/listen/wiki/Increasing-the-amount-of-inotify-watchers for info on how to fix this.
      
      E: Error building project: Step failed with status code 1.

    E: Error: Unable to build application, aborting.


To git.eu-3.platform.sh:222f5uv3bfma2.git
 * [new branch]      master -> master
Branch 'master' set up to track remote branch 'master' from 'platform'.

```

**Comment by Crell** [comment]
This is now ready for review.

**Pull request merged**